country_plan/migrations/0006_alter_countryplan_created_by_and_more.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.2.13 on 2024-05-19 13:47
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('country_plan', '0005_auto_20230207_0840'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='countryplan',
+            name='created_by',
+            field=models.ForeignKey(blank=True, editable=False, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='%(class)s_created_by', to=settings.AUTH_USER_MODEL, verbose_name='Created By'),
+        ),
+        migrations.AlterField(
+            model_name='countryplan',
+            name='updated_by',
+            field=models.ForeignKey(blank=True, editable=False, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='%(class)s_updated_by', to=settings.AUTH_USER_MODEL, verbose_name='Updated by'),
+        ),
```

Comment:
Did you forget to push the changes on models?

Reply:
What changes? I rebased develop and gave a manage.py makemigrations. Maybe these changes, what you mean, happened later.

Reply:
I was wondering how migration file were created? Did me miss any changes on models.py?

Reply:
Some changes were simply a consequence of the new Django behaviour.
You can try it on your own env: a simple change to Django4.2 brings new model migrations. Was a surprise for me, too.